parser/unparse.go

```diff
@@ -348,8 +365,15 @@ func (s *BlockStmt) String() string {
 			}
 		}
 
+		insideHeredoc := false
 		for i := 1; i < len(stmts); i++ {
-			if len(stmts[i]) > 0 {
+			if stmts[i] == "<HLB-RAW-HEREDOC-START>" || stmts[i] == "<HLB-RAW-HEREDOC-END>" {
```

Comment:
Seems like we only need one marker? Can we move this to a constant at the top of this package. Maybe `<HLB-HEREDOC-MARKER>`.

Reply:
fixed.